docs/install.md

```diff
@@ -106,3 +107,34 @@ That way you have control over when a breaking change affects you.
     # │        │        │        │        │        │        │        │        │        │        │
     # (1, 0)───(1, 1)───(1, 2)───(1, 3)───(1, 4)───(1, 5)───(1, 6)───(1, 7)───(1, 8)───(1, 9)───(1, 10)
     ```
+
+
+### Installing on Docker
+
+This will create a Docker image with the source code cloned from Github. This setup can be practical for development when you don't want to change any existing installation on your host machine.
+
+0. On your host sytem docker needs to be installed.
```

Comment:
Nit: Change to "Install Docker on your host system" with a link to the Docker installation page: https://docs.docker.com/install/#supported-platforms